src/compiler/program.ts

```diff
@@ -226,6 +226,11 @@ namespace ts {
         for (const diagnostic of diagnostics) {
             output += formatDiagnostic(diagnostic, host);
         }
+
+        output += host.getNewLine();
+        output += `Compilation finished - ${diagnostics.length} issue${diagnostics.length > 1 ? "s" : ""}`;

```

Comment:
This needs to be localized (so you'll need a new entry, or to reuse an existing entry) in `diagnosticMessages.json`)

Reply:
And also this needs to use same path and message as watch implementation

Reply:
Thanks for the feedback! I’ll get to those changes tomorrow 👍

Reply:
Okay! I've got those changes pushed in 
028b8671f7ab1a9de3f3fdf4c5bd312b2b130d7f. 

@sheetalkamat I didn't quite understand what you meant by "path and message", so I looked at the watch implementation. It was really helpful, thanks for pointing me in that direction! I couldn't re-use the exact same message (since it refers to being in watch mode and this is for normal compilation) but I used a very similar implementation and message. Let me know what I can do to make it fit in with the rest of the codebase.